atest/testdata/standard_libraries/builtin/verify.robot

```diff
@@ -68,6 +68,13 @@ Should Not Be Equal
     ${STR1}    ${INT1}
     ${STR1}    1
 
+Should Not Be Equal With Case Insensitivity
+    [Documentation]     FAIL foo == foo
```

Comment:
We need to decide should this and other similar errors be reported like this (i.e. `foo == foo`) or should we rather show the original values and mention that comparison was case-insensitive (e.g. `foo == FOO (case-insensitively)`). Probably better to discuss that in the PR comments or perhaps even in the comments of the original issue.